outbound-dialer/src/main/java/org/motechproject/nms/outbounddialer/domain/AuditRecord.java

```diff
@@ -8,35 +8,62 @@
 
 @Entity(tableName = "nms_obd_audit_records")
 public class AuditRecord {
+    /**
+     * The identifier field is used in each targetFile csv row combined with the subscription id to form a uniquely
+     * identifying id for a specific OBD record
+     */
     @Field
     @UIDisplayable(position = 0)
-    private FileType type;
+    private String identifier;
```

Comment:
I think I understand it now, looking at the code further down. Can we call this something else like fileId or batchId? Mostly a nitpick though